extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -269,26 +269,17 @@ export class RefreshManager {
   }
 
   /**
-   * Terminates the current refresh cycle, if one currently exists. Returns
-   * true if the cycle was canceled successfully, false otherwise.
-   *
-   * @return {boolean}
-   */
-  stopRefreshCycle() {
-    if (this.refreshTimeoutId_) {
-      this.timer_.cancel(this.refreshTimeoutId_);
-      this.refreshTimeoutId_ = null;
-      return true;
-    }
-    return false;
-  }
-
-  /**
-   * Converts config to appropriate units, modifying the argument in place.
+   * Converts config to appropriate units, modifying the argument in place. This
+   * also ensures that visiblePercentageMin is in the range of [0, 100].
    * @param {!RefreshConfig} config
    * @return {!RefreshConfig}
    */
-  convertConfiguration_(config) {
+  convertAndSanitizeConfiguration_(config) {
+    if (config['visiblePercentageMin'] < 0 ||
+        config['visiblePercentageMin'] > 100) {
+      dev().error(TAG,
```

Comment:
Small correction: the refresh interval is set by the publisher. Everything else is from the network.